src/config/firebase-config.js

```diff
@@ -0,0 +1,16 @@
+import firebase from "firebase";
+
+const firebaseConfig = {
+  apiKey: "AIzaSyAOJRaxF3xBfHAhcTFvCCqdhiID6T74Sk8",
```

Comment:
Are you sure that this is not supposed to be secret?

Reply:
okay, removed and added in .env